### PR TITLE
Fix certificate emails

### PR DESCRIPTION
**Pull request by PineappleIOnic**
## What does this PR do?

This PR fixes certificate emails, adding appwrite branding while I'm there.

BEFORE:
image

AFTER:
image

## Checklist

- [X] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [ ] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: app/config/locale/translations/en.json [hunk 1]
```diff
@@ -43,6 +43,12 @@
     "emails.invitation.footer": "If you are not interested, you can ignore this message.",
     "emails.invitation.thanks": "Thanks",
     "emails.invitation.signature": "{{project}} team",
+    "emails.certificate.subject": "Certificate failure for %s",
```
Comment (Meldiron) [inline app/config/locale/translations/en.json:46]: Hmm, I think we had a script to generate translations for all languages, from EN. using AI. We could run it.

File: src/Appwrite/Platform/Workers/Certificates.php [hunk 2]
```diff
@@ -439,40 +439,22 @@ private function notifyError(string $domain, string $errorMessage, int $attempt,
...
-            'domain' => $domain,
-            'error' => '<br><pre>' . $errorMessage . '</pre>',
-            'attempt' => $attempt,
-            'project' => 'Console',
-            'redirect' => 'https://' . $domain,
         ];
 
         $queueForMails
-            ->setSubject($subject)
+            ->setSubject('Certificate failed to generate')
```
Comment (Meldiron) [inline src/Appwrite/Platform/Workers/Certificates.php:454]: We should use locale for subject too

File: app/config/locale/translations/en.json (hunk 1 above)
Comment (Meldiron) [reply app/config/locale/translations/en.json:46]: https://github.com/appwrite/appwrite/pull/7442

We might need to check with CHristy for thisPR to be merged beforehand

Comment (PineappleIOnic) [reply app/config/locale/translations/en.json:46]: Do you know where this script is?

**Review by Meldiron — APPROVED** [review]

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**